backend/oasst_backend/models/user.py

```diff
@@ -44,6 +45,27 @@ class User(SQLModel, table=True):
     # terms of service acceptance date
     tos_acceptance_date: Optional[datetime] = Field(sa_column=sa.Column(sa.DateTime(timezone=True), nullable=True))
 
+    _most_used_lang: Optional[str] = None
+
+    @classmethod
+    def most_used_lang(self, session: Session):
+        print(self._most_used_lang)
+        print("UserSM : ", self.username, self._most_used_lang)
+        if self._most_used_lang is not None:
+            return self._most_used_lang
+
+        query = session.query(Message.lang, sa.func.count(Message.lang).label("total"))\
```

Comment:
It will be very expensive to run this for all users (e.g. the update user-stats queries take minutes already). Maybe it could  be noted somewhere that this is only for filtering the trollboard.